.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,91 @@
+# docs https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms
+# https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema
+
+name: Create a bug report
+description: Errors and regression reports with complete reproducing test cases and/or stack traces.
+labels: [requires triage]
+body:
+  - type: markdown
+    attributes:
+      value: "
+
+**FOR ALL USERS - PLEASE READ THIS PARAGRAPH AS IT HELPS US TO DO OUR WORK MORE EFFICIENTLY:**
+
+The **VAST MAJORITY** of bugs reported are in fact
+**USAGE QUESTIONS, NOT BUGS** for APIs that can take some practice to familiarize with,
+and which are also tailored towards high quality relational designs that
+beginners are sometimes less familiar with.
+
+TODO
+"
+  - type: markdown
+    attributes:
+      value: "
+
+**GUIDELINES FOR REPORTING BUGS**
+
+Your bug report should include the following features:
+
+1. a **succinct description of the problem** - typically a line or two at most
+
+2. **succinct, dependency-free code which reproduces the problem**, otherwise known as a [Minimal, Complete, and Verifiable](https://stackoverflow.com/help/mcve) example.
```

Comment:
```suggestion
2. **succinct code example which reproduces the problem**, otherwise known as a [Minimal, Complete, and Verifiable](https://stackoverflow.com/help/mcve) example.
```